dynamic.js

```diff
@@ -281,14 +282,17 @@ module.exports = function (fastify, opts, next) {
         const rawJsonSchema = fastifyResponseJson[key]
         const resolved = ref.resolve(rawJsonSchema)
 
+        /* istanbul ignore next */
         if (resolved.type || resolved.$ref) {
           responsesContainer[key] = {
             schema: resolved
           }
         } else {
+          /* istanbul ignore next */
           responsesContainer[key] = resolved
         }
 
+        /* istanbul ignore next */
```

Comment:
We'll never have a `description` unless we directly specify it before the `if`

Reply:
let's drop the if.

Reply:
Ok then